.github/ISSUE_TEMPLATE/work-item-template.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Work item template
+about: This issue describes a work-item ....
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+- describe purpose of work-items
+- label issue with affected components
```

Comment:
maybe also add
- link to a feature or bug (if applicable)

Reply:
makes sense, will add ...